erpnext/hr/doctype/employee/employee.json

```diff
@@ -726,13 +726,15 @@
    "fieldtype": "Int",
    "hidden": 1,
    "label": "lft",
+   "no_copy": 1,
```

Comment:
This should go to framework right? It never makes sense to copy lft/rgt.

Reply:
The framework already handles this while adding nested set fields with no copy enabled when you enable **Is Tree** for doctypes

https://github.com/frappe/frappe/blob/76cc6bf10b6c981dd4c0e0d7e86a28937ce4bb06/frappe/core/doctype/doctype/doctype.py#L629-L651

Earlier making tree view doctypes wasn't configurable and these fields were manually added in most of the doctypes. Same with Employee.

Will disable no copy for other tree doctypes in ERPNext too.

Reply:
Fixed in copy doc: https://github.com/frappe/frappe/pull/15128

But merging this anyway.